<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorDialog</class>
 <widget class="QDialog" name="MonitorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Monitor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <property name="spacing">
    <number>10</number>
   </property>
   <item row="2" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="lineEndingOptions"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QPlainTextEdit" name="outputText"/>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lineEndingLabel">
     <property name="text">
      <string>Line ending</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="sendButton">
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLineEdit" name="inputText"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
